src/main/java/matching_system/MatchManager.java

```diff
@@ -0,0 +1,36 @@
+package matching_system;
+
+import data_access_storage.RequestModel;
+import data_access_storage.UserRepoManager;
+import entities.Matcher;
+import entities.UserAccount;
+
+import java.util.ArrayList;
+
+public class MatchManager implements MatcherInputBoundary{
+    private UserAccount user;
+    private Matcher match;
+
+    final UserRepoManager userRepoManager;
+
+    public MatchManager(MatchRequestModel matchRequestModel, UserRepoManager userRepoManager){
```

Comment:
Following clean architecture,  MatchManager should uses OutputBoundary which is an interface implemented by presenter.  We need an instance variable that is the OutputBoundary.